backend/endpoints/jokes/blueprint.py

```diff
@@ -1,11 +1,68 @@
-from flask import Blueprint, make_response
+from flask import Blueprint, make_response, request
+from utils.myclass import EelJokes, db
+import json
 
 bp = Blueprint(
     "eelbot_joke",
     __name__,
     url_prefix="/joke",
 )
 
-@bp.route("/")
-def main():
-    return make_response(("Hello Eels!", 200))
\ No newline at end of file
+# @bp.route("/")
+# def main():
+#     return make_response(("Hello Eels!", 200))
+
+@bp.route("/", methods=['POST'])
+def create():
+    json_data = request.json
+    new_task = EelJokes(JokeText=json_data[0], JokeTextLine2=json_data[1])
+
+    try:
+        db.session.add(new_task)
+        db.session.commit()
+        return "Joke added successfully"
+    except:
+        return "There was an issue adding your joke"
+    pass
+
+@bp.route("/<int:num_jokes_per_page>/page/", methods=['GET'])
+def pagenum(num_jokes_per_page):
+    import math
+    rows = EelJokes.query.count()
+    num_pages = math.ceil(rows/num_jokes_per_page)
+    return str(num_pages)
+
+@bp.route("/<int:num_jokes_per_page>/page/<int:page>", methods=['GET'])
```

Comment:
pagination seems overkill at the scale we work at.

regardless: pagination should be via query parameters not path segments

Reply:
Done